libntcan/CMakeLists.txt

```diff
@@ -47,22 +43,10 @@ add_dependencies(ntcan build_ntcan3)
 add_dependencies(ntcan build_ntcan317)
 
 
-
-
 install(PROGRAMS common/lib/${MYARCH}/libntcan.so common/lib/${MYARCH}/libntcan.so.3 common/lib/${MYARCH}/libntcan.so.3.1.7
```

Comment:
Well, those are no CMake Targets. Those are precompiled libraries... Would this work with TARGETS?